pkgs/top-level/aliases.nix

```
@@ -1010,6 +1010,7 @@ mapAliases {
   licensor = throw "'licensor' has been removed due to lack of upstream maintenance"; # Added 2025-01-25
   lightdm_gtk_greeter = lightdm-gtk-greeter; # Added 2022-08-01
   lightstep-tracer-cpp = throw "lightstep-tracer-cpp is deprecated since 2022-08-29; the upstream recommends migration to opentelemetry projects.";
+  lima-bin = lima; # Added 2025-05-13
```

Maybe we could show a message to let users know that it has been replaced by `lima`, rather than replacing it silently?
```suggestion
  lima-bin = lib.warnOnInstantiate "lima-bin has been replaced by lima" lima; # Added 2025-05-13
```

```console
> nix build .#lima-bin
evaluation warning: lima-bin has been replaced by lima
```